lib/ace/keyboard/vim.js

```diff
@@ -3163,6 +3170,8 @@ dom.importCssString(".normal-mode .ace_cursor{\
         var height = cm.listSelections().length;
         if (insertAt == 'eol') {
           head = Pos(head.line, lineLength(cm, head.line));
+        } else if (insertAt == 'bol') {
```

Comment:
not entirely clear what `bol` stands for

Reply:
it is beginning of line similar to eol, a bit confusing but used elsewhere in this file, also in other editors, e.g. command names of sublime.